pdf_report/extract_pdf.py

```diff
@@ -28,16 +28,21 @@ def get_cursor(conn: connection) -> cursor:
     return conn.cursor(cursor_factory=RealDictCursor)
 
 
+def query_db(conn: connection, query: str) -> list[tuple]:
```

Comment:
I like the use of a function to run queries, helps reduce dry code

Reply:
thank you